etc/bash-utils.sh

```diff
@@ -296,12 +318,35 @@ build-rmm-cpp() {
 
 export -f build-rmm-cpp;
 
+configure-kvikio-cpp() {
+    config_args="$@"
+    update-environment-variables $@ >/dev/null;
+    config_args="-D rmm_ROOT=${RMM_ROOT}
```

Comment:
libkvikio doesn't use librmm, does it? Can we remove `rmm_ROOT`?

Reply:
Good point, yeah I think we can.